makefiles/default-channel.inc.mk

```diff
@@ -1,13 +1,17 @@
 # Set a custom channel if needed
-ifneq (,$(filter cc110x,$(USEMODULE)))          # radio is cc110x sub-GHz
-  DEFAULT_CHANNEL ?= 0
+ifneq (,$(DEFAULT_CHANNEL))
+ifneq (,$(filter cc110x,$(USEMODULE)))        # radio is cc110x sub-GHz
```

Comment:
Please fixe the indentation of this block (two spaces before)

Reply:
Ok.